### PR TITLE
moves squidex schema to crn folder

### PR DESCRIPTION
**Pull request by markcsims**
https://asaphub.atlassian.net/browse/GP2-43

- [ ] Setup dev 
- [ ] Setup prod apps
- [x] Use user json schema 
- [ ] User webhook rule from CRN and add it to gp2 app
- [ ] Set up the pipeline to sync the schema for gp2 into dev 
- [ ]  prod applications on master merge.
- [ ] verify schema job
- [x] SQUIDEX_CLIENT_SECRET and SQUIDEX_SHARED_SECRET
- [x] test backup
- [x] test algolia sync
- [x] test create environment

### CODE REVIEW

File: .github/environment/Production [hunk 1]
```diff
@@ -16,6 +16,10 @@ react-app-gtm-container-id=GTM-M5HZBRZ
...
 ses-region=us-east-1
 sls-stage=production
-squidex-app-name=asap-hub
 squidex-backup-bucket-name=asap-hub-production-data-backup
-squidex-client-id=asap-hub:default
+crn-squidex-app-name=asap-hub
+crn-squidex-ci-client-id=asap-hub:default
+crn-squidex-api-client-id=asap-hub:default
+gp2-squidex-app-name=gp2-hub
+gp2-squidex-ci-client-id=gp2-hub:default
```
Comment (peterstarling) [inline .github/environment/Production:24]: didn't we say we would use different clients for API and CI for GP2?

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: .github/environment/Production (hunk 1 above)
[the remark this answers is not in the document]
Comment (markcsims) [reply .github/environment/Production:24]: yes, but we don't have production yet. They are more placeholders.

Comment (markcsims) [reply .github/environment/Production:24]: yes, but we don't have production yet. They are more placeholders.

File: .github/scripts/squidex/create-app.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import requests
 import os
 import sys
 
+import requests
```
[the remark this answers is not in the document]
Comment (markcsims) [reply .github/scripts/squidex/create-app.py:4]: no idea, it was there before. I think vscode moved it.

**Pull request merged**